src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -204,95 +211,18 @@ private class TaskLauncherActor(
     case RecheckIfBackOffUntilReached => OfferMatcherRegistration.manageOfferMatcherStatus()
   }
 
-  private[this] def receiveTaskLaunchNotification: Receive = {
```

Comment:
@meichstedt, I don't really know the purpose of the op rejections and time outs. What's the idea here? I think we do have to model rejection by Mesos, though.